arch/x86/include/asm/kvm_host.h

```diff
@@ -1324,6 +1324,43 @@ enum kvm_apicv_inhibit {
 	__APICV_INHIBIT_REASON(SEV),			\
 	__APICV_INHIBIT_REASON(LOGICAL_ID_ALIASED)
 
+struct pkvm_memcache {
+	struct pkvm_mem_range {
+		phys_addr_t addr;
+		size_t size;
+	} mem_range;
```

Comment:
Maybe still name this field `head` (like in 6.12 and on ARM) instead of `mem_range`, to emphasize that it is still a linked list?

Reply:
Sounds good to me.